### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by timifasubaa**
First pass of making the codebase more pep8 compliant. 

@mistercrunch

### CODE REVIEW

File: superset/__init__.py [hunk 1]
```diff
@@ -20,14 +20,12 @@
 from superset.connectors.connector_registry import ConnectorRegistry
 from superset import utils, config  # noqa
 
-
```
Comment (xrmx) [inline superset/__init__.py]: Doesn't look like an improvement to me

**Review by xrmx — COMMENTED** [review]
Looks like a lot of whitespace changes for not much gain to me, there's not much consistency too. The s/type/isinstance change looks good though.

**Comment by mistercrunch** [comment]
LGTM

**Comment by timifasubaa** [comment]
Removed changes whose previous state was already pep8 compliant  or I thought did not bring much benefit.

**Pull request merged**